src/main/java/Duke.java

```diff
@@ -1,10 +1,269 @@
+            if(!dukeTextFile.exists()){
+                try {
+                    dukeTextFile.createNewFile();
+                } catch (Exception e){
+                    e.printStackTrace();
+                }
+            } else {
+                loadDukeTextFile(dukeTextFile ,list );
+            }
+    }
+
+    public static void loadDukeTextFile(File dukeTextFile ,TaskList list) {
+        try {
+            Scanner dukeTextScanner = new Scanner(dukeTextFile);
+            while (dukeTextScanner.hasNextLine()) {
+                String dukeTextLine = dukeTextScanner.nextLine();
+                String taskLetter = dukeTextLine.substring(0,1);
+                if (taskLetter.equals("T")) { //ToDo
+                    String[] dukeLineSplit = dukeTextLine.split("\\|" ,3);
+                    String todoDescription = dukeLineSplit[2];
+                    Task task = new ToDo(todoDescription);
+                    list.addTaskList(task);
+                } else if (taskLetter.equals("D")) {//Deadline
+                    String[] dukeLineSplit = dukeTextLine.split("\\|" ,4);
+                    String deadlineDescription = dukeLineSplit[2].trim();
+                    String deadlineBy = dukeLineSplit[3].trim();
+                    Task task = new Deadline(deadlineDescription, deadlineBy);
+                    list.addTaskList(task);
+                } else {
+                    String[] dukeLineSplit = dukeTextLine.split("\\|" ,4);
+                    String eventDescription = dukeLineSplit[2].trim();
+                    String eventAt = dukeLineSplit[3].trim();
+                    Task task = new Event(eventDescription, eventAt);
+                    list.addTaskList(task);
+                }
+            }
+            writeDukeTextFile(list);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+
+    }
+
+    public static void writeDukeTextFile(TaskList list) {
+        File dukeTextFile = new  File("Duke.txt");
+        try {
+            FileWriter dukeTextWriter = new FileWriter(dukeTextFile);
+            int lineNumber = list.getTaskListCount();
+            for (int i = 0 ; i < lineNumber ; i++) {
+                dukeTextWriter.write(list.textString(i));
+            }
+            dukeTextWriter.close();
+        } catch (Exception e) {
+            e.getStackTrace();
+        }
+
+    }
+
+
+
+    public static void printGreeting() {
+        String logo = "      ____        _        \n"
+                + "     |  _ \\ _   _| | _____ \n"
+                + "     | | | | | | | |/ / _ \\\n"
+                + "     | |_| | |_| |   <  __/\n"
+                + "     |____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println(LINE_SEPARATOR);
+        System.out.println("    Hello! I'm Duke");
+        printCommandGuide();
+    }
+
+
+    public static void printCommandGuide(){
+        //prints list of commands
```

Comment:
I feel its quite clear from how you named the method, what it is doing. Hence, this comment might be unnecessary.